### DISCUSSION TITLE
Issue with Supabase Realtime Subscription in Docker and Cloud Environment

### DISCUSSION THREAD

**Discussion by Asaraspo** [supabase/supabase#15347] [Questions] [answered]
I am currently implementing a realtime feature in my react native app.
I have followed the documentation and activated the realtime feature for the public.confirmation table in my Supabase dashboard. However, when updating field values through the UI or via the javascript library, I do not receive any logged updates in my console. I tried this using a local Docker and the Cloud Environement. RLS is deactivated for now.

This is my javascript code:
```javascript
supabase
      .channel("supabase_realtime")
      .subscribe((status) => console.log(status));
```

And I am getting SUBSCRIBED logged whenever the function is invoked. 

The strange thing: I tried using the inspector: https://realtime.supabase.co/inspector and am getting updates there. 
Is this a problem with react native?

**Comment by GaryAustin1** [comment, marked as answer]
Not sure if you are only showing part of your code, but that is not enough to get database changes.  
See https://supabase.com/docs/guides/realtime/postgres-changes#schema-changes

**Reply by Asaraspo** [reply]
Thanks. Worked like a charm.
